src/components/ConnectedMap/map-libre-adapter/index.tsx

```diff
@@ -141,6 +142,10 @@ function MapboxMap(
       dispatchMetricsEvent(EVENT_MAP_IDLE, true);
     });
     setMap(mapInstance);
+    // use initial extent if no coords in url
+    if (!currentMapPosition && appConfig.extent) {
```

Comment:
i see `LAYERS_BY_DEFAULT` was removed from config but no `extent` was added. Should it be added or should it be on cd-repo config?